src/Common/Constants.ts

```diff
@@ -437,3 +437,8 @@ export class TerminalQueryParams {
   public static readonly SubscriptionId = "subscriptionId";
   public static readonly TerminalEndpoint = "terminalEndpoint";
 }
+
+export class LoadOfferRetry {

```

Comment:
Move this into where it is called. Trying to avoid adding to Constants for things that not widely used